internal/compile/compile.go

```diff
@@ -1597,9 +1597,9 @@ func (fcomp *fcomp) args(call *syntax.CallExpr) (op Opcode, arg uint32) {
 		fcomp.expr(kwargs)
 	}
 
-	// TODO(adonovan): avoid this with a more flexible encoding.
```

Comment:
The TODO should remain, as technically this is still a bug in the compiler.

Reply:
Done